erigon-lib/Makefile

```diff
@@ -1,7 +1,7 @@
 GOBINREL = build/bin
 GOBIN = $(CURDIR)/$(GOBINREL)
 
-BUILD_TAGS = nosqlite,noboltdb,disable_libutp
```

Comment:
@anacrolix i don't remember why we have `disable_libutp` - probably too much uncontrollable logging.

Reply:
I don't remember disabling it. The pure Go version is not good. I think libutp has been stable for a long time and the logging fixed. It's possible the tag doesn't do anything anymore, it seems to be enabled anyway for many users.

Reply:
removed